frappe/website/web_template/section_with_cards/section_with_cards.html

```diff
@@ -11,6 +11,12 @@
 }) -%}
 <div class="mt-6 {{ col_classes }}">
 	<div class="card {{ card_classes }} h-100">
+	    {%- if image -%}
+	    {{ frappe.render_template('templates/includes/image_with_blur.html', {
+			"src": image,
+			"class": ["card-img-top p-2"]
```

Comment:
Let's not use higher specificity classes that are hard to override from Website Theme. Better to add the padding in `page_builder.scss` file.

Reply:
Ok, thanks will do that.